src/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0 auto;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star.svg) center center no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 *:first-child {
+  background: url(images/star-active.svg) center center no-repeat;
+}
+
+.stars--2 *:nth-child(-n + 2) {
+  background: url(images/star-active.svg) center center no-repeat;
+}
+
+.stars--3 *:nth-child(-n + 3) {
+  background: url(images/star-active.svg) center center no-repeat;
+}
+
+.stars--4 *:nth-child(-n + 4) {
+  background: url(images/star-active.svg) center center no-repeat;
+}
+
+.stars--5 *:nth-child(-n + 5) {
+  background: url(images/star-active.svg) center center no-repeat;
```

Comment:
check this checklilst point
<img width="945" alt="image" src="https://user-images.githubusercontent.com/102411130/223366278-42e184dd-ab8c-46c3-ae96-7b772d989994.png">